### PR TITLE
Fix geodetic lines

### PR DESCRIPTION
**Pull request by pHill5136**
Fixed geodetic line tool for pro 1.4

### CODE REVIEW

**Comment by pHill5136** [comment]
@csmoore please review and merge.

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProLinesViewModel.cs [hunk 1]
```diff
@@ -58,11 +58,36 @@ public LineFromTypes LineFromType
...
+            get { return lineType;  }
+            set
+            {
+                lineType = value;
+                RaisePropertyChanged(() => LineType);
+                ResetFeedback();
+            }
+        }
+
+        public CurveType DeriveCurveType(LineTypes type)
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProLinesViewModel.cs]: Could also be called just plain old "LinesTypesToCurveType"

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProLinesViewModel.cs [hunk 2]
```diff
@@ -58,11 +58,36 @@ public LineFromTypes LineFromType
             }
         }
 
+        LineTypes lineType = LineTypes.Geodesic;
+        public LineTypes LineType
+        {
+            get { return lineType;  }
+            set
+            {
+                lineType = value;
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProLinesViewModel.cs]: This probably isn't called very often (but I'm not sure how the view even hooks up to this) so probably doesn't need a guard:
if (value == lineType) return

File: source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProTabBaseViewModel.cs [hunk 3]
```diff
@@ -821,8 +822,9 @@ private void SetGeodesicDistance(MapPoint p1, MapPoint p2)
             });
 
             ClearTempGraphics();
+            Geometry newline = GeometryEngine.GeodeticDensifyByLength(polyline, 0, LinearUnit.Meters, type);
```
Comment (csmoore) [inline source/DistanceAndDirection/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProTabBaseViewModel.cs]: Could maybe add a comment about what the default value ("0") does (I guess says the max segment length will be 50km: https://pro.arcgis.com/en/pro-app/sdk/api-reference/#topic8245.html )

**Comment by csmoore** [comment]
Some very minor feedback (just to prove I looked at) - LGTM - merging

**Pull request merged**